pkg/serveswagger/swagger_test.go

```diff
@@ -0,0 +1,28 @@
+package serveswagger
+
+import (
+	"os/exec"
+	"regexp"
+	"testing"
+)
+
+func TestCheckTimestamps(t *testing.T) {
+	exec.Command("go", "generate", "./cmd/go-swagger/restapi")
```

Comment:
This does nothing - the command is not executed. It needs to run, and be checked for error.

i.e.
```go
gen := exec.Command(/*...*/)
if err := gen.Run(); err != nil{
    //...
}